ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3204,4 +3204,8 @@
     @LogMessage(level = WARN)
     @Message(id = 515, value = "[EJB3.2 spec, section 4.9.2] Singleton session beans are not allowed to implement 'javax.ejb.SessionBean' interface. This interface on bean '%s' is going to be ignored and should be removed.")
     void singletonCantImplementSessionBean(String className);
+
+    @LogMessage(level = INFO)
+    @Message(id = 516, value = "IIOP bindings for session bean named '%s' in deployment unit '%s' are as follows: %s")
+    void iiopBindings(final String ComponentName, final String ModuleName, String Name);
```

Comment:
Just some minor style issues: either use final for all 3 params or no final.
`ComponentName` -> `componentName`